src/test/java/org/fcrepo/migration/InlineXmlIT.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.migration;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.io.FileUtils;
+import org.fcrepo.storage.ocfl.OcflObjectSessionFactory;
+import org.junit.After;
+import org.junit.Test;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * @author pwinckles
+ */
+public class InlineXmlIT {
+
+    private static final String OBJECT_ID = "info:fedora/1711.dl:CModelAudioStream";
+    private static final String AUDIT_ID = OBJECT_ID + "/AUDIT";
+    private static final String DC_ID = OBJECT_ID + "/DC";
+    private static final String DS_COMPOSITE_MODEL_ID = OBJECT_ID + "/DS-COMPOSITE-MODEL";
+    private static final String RELS_EXT_ID = OBJECT_ID + "/RELS-EXT";
+
+    private static final Map<String, String> EXPECTED_DIGESTS = Map.of(
+            AUDIT_ID, "c5aa5d74afc74aaf769b685e08d32cbc",
+            DC_ID, "9b3cb6287c11be2eddd3ff0a66805103",
+            DS_COMPOSITE_MODEL_ID, "84184d2d8ee6eae9dbcc5f02eaff681c",
+            RELS_EXT_ID, "c30c3df0877b8f113f8f4f844bdfe3e6"
+    );
+
+    private ConfigurableApplicationContext context;
+    private Migrator migrator;
+    private OcflObjectSessionFactory sessionFactory;
+
+    @After
+    public void tearDown() {
+        if (context != null) {
+            context.close();
+        }
+    }
+
+    private void setup(final String name) throws Exception {
+        // Create directories expected in this test (based on `spring/ocfl-user-it-setup.xml`)
+        final var storage = Paths.get(String.format("target/test/ocfl/%s/storage", name));
+        final var staging = Paths.get(String.format("target/test/ocfl/%s/staging", name));
+
+        if (Files.exists(staging)) {
```

Comment:
should this be storage instead of staging?

Reply:
yes...